localization/languages/pt-BR.json

```diff
@@ -64,8 +64,8 @@
         "viewTasks": "Ver tarefas",
         "newTask": "Nova tarefa", //"new" is a verb - it is used for a button at the bottom of the task overlay
         "defaultTaskName": "Tarefa %n", //this is the name used for newly-created tasks; %n is replaced with a number ("task 1", "task 2", etc)
-        "taskDeleteWarning": {"unsafeHTML": null}, //missing translation
-        "returnToTask": null, //missing translation
+        "taskDeleteWarning": {"unsafeHTML": "Tarefa excluir!"},
```

Comment:
"taskDeleteWarning"  -> can it just be "task deleted" or should it have "warning, Task deleted"?